src/utils/useItisSearch.js

```diff
@@ -59,5 +59,5 @@ export default function useItisSearch(searchKey, maxResults = 50) {
     if (searchKey) fetchData();
   }, [searchKey, maxResults]);
 
-  return { data, loading, error };
+  return { data, loading, error, setError, setLoading };
```

Comment:
The only time that I see `setError` being used is to clear the error when the Search button is clicked. It probably makes more sense to clear the error within `useItisSearch` so that it is always reset before a request is made.

Reply:
Could you elaborate on this one? I left a question about it on slack.